Motorcycle.js

```diff
@@ -1,6 +1,29 @@
 var Car = require('./Car.js');
 
 //Implement your Motorcycle here
-function Motorcycle() {};
+function Motorcycle(make, model, year, color, seats, passengers) {
+
```

Comment:
Here, I'd recommend calling ```Car.call``` only once, and then overriding the user input with a conditional:
```
Car.call(this, make, model, year, color, seats, passengers);
	if (this.seats > 2) {
		this.seats = 2;
	}
```